contents/uix/uix.md

```diff
@@ -45,3 +47,65 @@ Just like Software Development, UX Design is an iterative process. It involves m
+  
+### User Testing
+After Prototyping, the next stage is User Testing. This is where you would:
+  * Should have a clear sense of the user goals at this stage
+  * Use a low or high fidelity prototype during [usability testing](http://www.uxbooth.com/articles/usability-testing-dont-guess-test/) to determine product effectiveness
+  * Follow an [iterative design process](https://www.interaction-design.org/literature/article/design-iteration-brings-powerful-results-so-do-it-again-designer) to re-prototype after gathering user feedback
+  
+```Deliverables (before testing)```
+  * [Usability Test Plan](https://www.smileycat.com/wp-content/uploads/2016/05/usability-test-plan-dashboard.png) - Summary of background, goals, and test methodology
+  * [User Testing Script](http://www.ariadne.ac.uk/images/issue62-loureiroKoechlin/CeciliaLoureiroKoechlin-03-small.gif) - What to say as the facilitator, encouraging users to think out loud during the test
+  
+```Deliverables (after testing)```
+  * [Usability Reports](https://image.slidesharecdn.com/january2012nhupa-120206210831-phpapp02/95/delivering-results-how-do-you-report-user-research-findings-18-728.jpg?cb=1374224669) - Thorough introduction to what aspects of the product was or was not effective and how to improve
+  * [Journey Maps](https://cdn-images-1.medium.com/max/1600/1*jAfZXNWAx50fWEvoF-eWLw.png) - More detailed user flow based on how users are observed to use the product in actual testing conditions
+  * [Refined Prototype](https://cdn-images-1.medium.com/max/1600/1*8qPOD1DG19Kj2sfJkTk3Eg.png) - A more refined prototype that is closer to meeting user expectations, each iteration should always be tested before moving to a higher fidelity
+  
+### Maintenance
+After User Testing and the launch of the final product, the next stage is Maintenance. This is where you would:
+  * Follow-up with the product after the final product launch
+  * Should be prepared to recursively update, revise and maintain the content in the application
+  
+```Deliverables```
+  * [Sitemaps](http://www.kristenjoybaker.com/uploads/1/3/7/6/13760055/3563790_orig.jpg) - Depiction of how all pages of product interconnect
+  * [Taxonomies](http://blog.fuzzymath.com/wp-content/uploads/2015/09/UCAN_Taxonomy-1024x775.png) - List of vocabularies used in the content
+  * [Content Governance Plans](https://image.slidesharecdn.com/contentstrategyin2015-150706154016-lva1-app6891/95/slide-43-1024.jpg) - Complete strategy for updating, revising and maintaining all content within the product
+  
+### Ending Notes
+The clearly-stated objectives and deliverables of each stage cumulates to a well-defined target audience and a well-designed prototype needed to match the expectations of that audience. UX Design requires empathy in the users' shoes and effort to address their problems, however all that is worth it in the grand scheme of delighting or even impressing users when they use your applications.
```

Comment:
"The clearly-stated objectives and deliverables of each stage cumulates to" cumulate*, but perhaps you meant `culminate in`?

"delighting or even impressing" -> delighting and impressing? I don't think impressing people is separate from, or more difficult than delighting them